lib/theme/ColorInitializer.dart

```diff
@@ -0,0 +1,75 @@
+import 'package:flutter/material.dart';
+
+enum ColorInitializer {
+  primary,
+  primaryContainer,
+  onPrimary,
+  secondary,
+  secondaryContainer,
+  onSecondary,
+  background,
+  onBackground,
+  surface,
+  onSurface,
+  onError
+}
+
+validateForeground(bgColor, context) {
+  switch (bgColor) {
+    case ColorInitializer.primary:
+    case ColorInitializer.primaryContainer:
+    case ColorInitializer.secondary:
+    case ColorInitializer.secondaryContainer:
+    case ColorInitializer.surface:
+    case ColorInitializer.background:
+      return getForegroundColor(bgColor, context);
+    default:
+      return throw Exception("Invalid Theme Color");
+  }
+}
```

Comment:
we don't need this method anymore remove this use `getForegroundColor` method.

Reply:
Done